packages/gatsby-plugin-offline/src/prefetch-catcher.js

```diff
@@ -0,0 +1,6 @@
+self.addEventListener(`message`, async ({ data }) => {
+  if (data.type === `prefetch`) {
+    const cache = await caches.open(`runtimecache`)
```

Comment:
need to figure out the proper name of cache

Reply:
You can look in the Chrome "application" tab for the name of the cache. Also probably worth looking through the sw-precache docs to see if they talk about this use case of explicitly caching resources from the client.

Reply:
i checked and couldn't find anything, workbox docs have it and wonder if it's the same, at worst case i can dig through source code or look at application tag